README.md

```diff
@@ -18,6 +18,7 @@ sensor:
 | **platform** | yes | | The sensor platform name.
 | **enable_notification** | no | `true` | Turn on/off `persistant_notifications` when a new IP is detected, can be `true`/`false`.
 | **exclude** | no | | A list of IP addresses you want to exclude.
+| **exclude_clients** | no | | A list of client_ids you want to exclude. For example, when using Alexa to connect to HA, you'll probably want to ignore pitangui.amazon.com (for North America), layla.amazon.com (EU/India) or alexa.amazon.co.jp (Far East), rather than IPs, of which there are hundred.

```

Comment:
This should not show specific clients, but instructions on how to get the clients

Reply:
My problem is that I don't actually know how these get set by HA ;-). From the name, I'd have expected a user-agent or something, but it seems like it's really a hostname

Reply:
Yes, clients are hostnames (or entrypoints), which there currently is no easy way of getting by the looks of it. (you need to manually dig through storage files, which no one should do), because of that, I'm not sure I want this in there.
at least not promoted.

Remove the readme entry and we can add it as a hidden (not supported) option.